packages/ui/src/shared/ui-settings.ts

```diff
@@ -9,8 +9,8 @@ export interface UISettings {
     uploadTableMaxSize: number;
     uploadTableUseLocalmode?: boolean;
 
-    uploadTableExcelBaseUrl?: string;
-    exportTableBaseUrl?: string;
+    uploadTableExcelBaseUrl?: string | Record<string, string>;
+    exportTableBaseUrl?: string | Record<string, string>;
```

Comment:
Probably it is better to use `clusters-config.json` to provide cluster specific options, something like:
```ts
interface ClusterConfig {
    ...
    uiSettings?: Partial<Pick<UISettings, 'uploadTableExcelBaseUrl' | 'exportTableBaseurl'>>;
}
```

Reply:
Then we can have simple function to get cluster-specific uiSettings, like:
```ts
function getClusterSpecificUISettings(clusterConfig?: ClusterConfig) {
    return {...uiSettings, ...clusterConfig.uiSettings};
}
```